src/GattInterface.h

```diff
@@ -164,10 +165,11 @@ struct GattInterface : DBusInterface
 	//
 	//     const char *pTextString = self.getDataPointer<const char *>("text/string", "");
 	template<typename T>
-	T getDataPointer(const char *pName, const T defaultValue) const
+	T getDataPointer(const char *pName, size_t& size) const
 	{
-		const void *pData = TheServer->getDataGetter()(pName);
-		return nullptr == pData ? defaultValue : static_cast<const T>(pData);
+		const void *pData = TheServer->getDataGetter()(pName, size);
```

Comment:
I would still keep a check that `pData` is not null. Or are we 100% sure it can not happen?